### DISCUSSION TITLE
Gaps in accessible and useable data

### DISCUSSION THREAD

**Discussion by MayteTDGeograma** [gopeg/issues-barriers#54] [General]
Despite INSPIRE allows the limitation of public access to spatial data sets and services, when the public access would adversely affect international relations, public security, or national defence -Article 13 (1) (b)-, this restriction should be limited to very few cases to avoid an important hindrance in the data access.

Analogously, when the data is provided under unclear licensing terms or are required fees, barriers are created in the use of INSPIRE datasets. To smooth these difficulties, fee waivers and the use of standard licences (e.g. CC By), to inform about the conditions of access and use, would be desirable.

**Comment by thorsten-reitz** [comment]
Isn't this more of a licensing and acess conditions issue than one of spatial continuity (e.g. for lack of consistent scales)?

**Comment by AndersOstmanNovogit** [comment]
This issue is included in #33 and reported in section 4.2.6 of D4.3.

**Reply by AndersOstmanNovogit** [reply]
Now included in D4.3 section 4.3.3
